src/pages/about/index.js

```diff
@@ -0,0 +1,247 @@
+                    flexDirection: "column",
+                    padding: "16px 0 0",
+                  }}
+                >
+                  <Card.Meta title="Sam Carmichael" description="Design" />
+                  <p className={styles.prev}>
+                    Previously:
+                    <br /> Product design manager @ WeWork, Founder of The
+                    UpStanding Desk
+                  </p>
+                  <div className={styles.social}>
+                    <a
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      href="https://linkedin.com/in/samcarmichael"
+                    >
+                      <LinkedinFilled />
+                    </a>
+                    <a
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      href="https://twitter.com/sccarmichael"
+                    >
+                      <TwitterSquareFilled />
+                    </a>
+                    <a
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      href="https://sam-carmichael.com"
+                    >
+                      <HomeFilled />
+                    </a>
+                  </div>
+                </Card>
+              </Col>
+              <Col xs={12} md={6}>
+                <Card
+                  cover={<img src={useBaseUrl("img/michael_teevan.png")} />}
+                  className={styles.personCard}
+                  bordered={false}
+                  headStyle={{}}
+                  bodyStyle={{
+                    display: "flex",
+                    flexGrow: "1",
+                    flexDirection: "column",
+                    padding: "16px 0 0",
+                  }}
+                >
+                  <Card.Meta title="Michael Teevan" description="Sales" />
+                  <p className={styles.prev}>
+                    Previously:
+                    <br /> Sales @ Clearbit, imgix, Nexonia / Certify
+                  </p>
+                  <div className={styles.social}>
+                    <a
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      href="https://linkedin.com/in/sambauch"
+                    >
+                      <LinkedinFilled />
+                    </a>
+                    <a
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      href="https://linkedin.com/in/sambauch"
+                    >
+                      <GithubFilled />
+                    </a>
+
+                    <a
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      href="https://linkedin.com/in/sambauch"
+                    >
+                      <TwitterSquareFilled />
+                    </a>
+                  </div>
```

Comment:
@mike6838 could you fill in here in this suggestion block your social links, and remove whatever you dont want to include?
```suggestion
                  <div className={styles.social}>
                    <a
                      target="_blank"
                      rel="noopener noreferrer"
                      href="https://linkedin.com/in/sambauch"
                    >
                      <LinkedinFilled />
                    </a>
                    <a
                      target="_blank"
                      rel="noopener noreferrer"
                      href="https://linkedin.com/in/sambauch"
                    >
                      <GithubFilled />
                    </a>

                    <a
                      target="_blank"
                      rel="noopener noreferrer"
                      href="https://linkedin.com/in/sambauch"
                    >
                      <TwitterSquareFilled />
                    </a>
                  </div>
```